core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -537,6 +539,60 @@ public T proxyDetector(ProxyDetector proxyDetector) {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Provides a service config to the channel.
+   *
+   * @param serviceConfig A nested map representing a Json object in the most natural way:
```

Comment:
Don't include the table in the `@param`. Move that up to the general description. `@param` should be short, like 2 sentences max, and ideally just a sentence fragment.

Reply:
Done.